modules/cnv-uploading-local-disk-image-dv.adoc

```diff
@@ -16,22 +16,24 @@ compressed by using `xz` or `gz`.
 * The `kubevirt-virtctl` package must be installed on the client machine.
 * The client machine must be configured to trust the {product-title} router's
 certificate.
-* A spare DataVolume that is the same size or larger than the disk that you are uploading.
 
 .Procedure
 
 . Identify the following items:
-** File location of the VM disk image that you want to upload
-** Name of the DataVolume
+** The name of the upload DataVolume that you want to use. If this DataVolume
+does not exist, it is created automatically.
+** The file location of the virtual machine disk image that you want to upload.
 
-. Run the `virtctl image-upload` command to upload your disk image.
-You must specify the DV name and file location. For example:
+. Upload the disk image by running the `virtctl image-upload` command.
+Specify the name and file location that you identified in the previous step.
+For example:
 +
 ----
-$ virtctl image-upload --dv-name=<upload-datavolume> --image-path=</images/fedora.qcow2> <1> <2>
+$ virtctl image-upload dv <datavolume_name> \ <1>
```

Comment:
When creating DataVolume, the size must be specified. --size=xxx

Reply:
Can we add `virtctl image-upload -h to get more`?

Reply:
@qwang1 Yes, I can add those. Is the size in GiB?

Reply:
> @qwang1 Yes, I can add those. Is the size in GiB?

The size can be in Gi, Mi, Ki, G, M, K (quantities must match the regular expression '^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$')

Reply:
> Gi, Mi, Ki, G, M, K (quantities must match the regular expression '^([+-]?[0-9.]+)([eEinumkKMGTP]_[-+]?[0-9]_)$')

@qwang1 Sorry for my ignorance, but I'm having trouble parsing the regex :) Can you give an example of a string that would be acceptable? Thanks.

Reply:
@qwang1 Please also review my latest commit (adding size info) when you get the chance: https://github.com/openshift/openshift-docs/pull/21113/commits/32f07a9a10ca0af311ec9b70c8f0ac4961fff8bf Thanks!

Reply:
@ousleyp Sorry the explanation in parentheses might mislead you. I figure that we can delete “The value must match the following regular expression: ^([+-]?[0-9.]+)([eEinumkKMGTP][-+]?[0-9])$”, just make the size unit clear. It can be in Mebibyte (MiB), Gibibyte (GiB)... or Megabyte (MB), Gigabyte (GB)... For example: --size=10Gi, 500Mi, 1G, 400M...I believe you have better description than me. 
Another thing is when uploading a image to an existing DV, `--no-create` should be specified. 
Here is the whole help of `virtctl image-upload` command for reference:
```
Upload a VM image to a DataVolume/PersistentVolumeClaim.

Usage:
  virtctl image-upload [flags]

Examples:
  # Upload a local disk image to a newly created DataVolume:
  virtctl image-upload dv dv-name --size=10Gi --image-path=/images/fedora30.qcow2

  # Upload a local disk image to an existing DataVolume
  virtctl image-upload dv dv-name --no-create --image-path=/images/fedora30.qcow2

  # Upload a local disk image to an existing PersistentVolumeClaim
  virtctl image-upload pvc pvc-name --image-path=/images/fedora30.qcow2

  # Upload to a DataVolume with explicit URL to CDI Upload Proxy
  virtctl image-upload dv dv-name --uploadproxy-url=https://cdi-uploadproxy.mycluster.com --image-path=/images/fedora30.qcow2

Flags:
      --access-mode string       The access mode for the PVC. (default "ReadWriteOnce")
      --block-volume             Create a PVC with VolumeMode=Block (default Filesystem).
  -h, --help                     help for image-upload
      --image-path string        Path to the local VM image.
      --insecure                 Allow insecure server connections when using HTTPS.
      --no-create                Don't attempt to create a new DataVolume/PVC.
      --pvc-name string          DEPRECATED - The destination DataVolume/PVC name.
      --pvc-size string          DEPRECATED - The size of the PVC to create (ex. 10Gi, 500Mi).
      --size string              The size of the DataVolume to create (ex. 10Gi, 500Mi).
      --storage-class string     The storage class for the PVC.
      --uploadproxy-url string   The URL of the cdi-upload proxy service.
      --wait-secs uint           Seconds to wait for upload pod to start. (default 60)

Use "virtctl options" for a list of global command-line options (applies to all commands).
```

Reply:
Thanks @qwang1! I just added size examples to the callout below the code block. I also changed the existing note to include info about how to change the command if you don't want to create a new DV, and added an entry to the virtctl commands module to show the same. Please let me know what you think when you get the chance.

Reply:
@ousleyp LGTM, thanks!